cmd/explaintest/r/explain_easy.result

```diff
@@ -79,9 +79,9 @@ TopN_7	1.00	root	Column#2:asc, offset:0, count:1
     └─TableScan_13	10000.00	cop[tikv]	table:t2, range:[-inf,+inf], keep order:false, stats:pseudo
 explain select * from t1 where c1 > 1 and c2 = 1 and c3 < 1;
 id	count	task	operator info
-IndexLookUp_11	1.11	root	
+IndexLookUp_11	11.08	root	
 ├─IndexScan_8	33.33	cop[tikv]	table:t1, index:c2, range:(1 1,1 +inf], keep order:false, stats:pseudo
-└─Selection_10	1.11	cop[tikv]	lt(Column#3, 1)
+└─Selection_10	11.08	cop[tikv]	lt(Column#3, 1)
```

Comment:
This change is a bug related with statistics.
Since we take the `RowID` column into consideration when the column of index are all be used.
But when calculating the selecitivity. https://github.com/pingcap/tidb/blob/master/statistics/selectivity.go#L215 Here we doesn't consider the `RowID` column.
So the rowid column is calculated twice, which makes the final row count smaller.

I've tried but it's not very easy to fix. So we can make this pr merged first. Then i'll try to fix it in another pr.